pkg/controller/metrics/metrics.go

```diff
@@ -197,6 +224,35 @@ type Calculator struct {
 func (mc *Calculator) Start(ctx context.Context) error {
 	log.Info("started metrics calculator goroutine")
 
+	// Register optional metrics and add them to corresponding maps, so controllers logging them can access
+	// the information
+	hiveConfig := &hivev1.HiveConfig{}
```

Comment:
I wanted to move this part where we register provision metrics (when we add to manager) but I think there's no hiveconfig object available to list then